detox/local-cli/testCommand/TestRunnerCommand.js

```diff
@@ -37,6 +42,15 @@ class TestRunnerCommand {
     let runsLeft = 1 + this._retries;
     let launchError = null;
 
+    if (this._startCommands.length > 0) {
+      try {
+        await Promise.race([sleep(1000), ...this._startCommands.map(cmd => cmd.execute())]);
```

Comment:
Debatable place. I decide here to wait for an extra 1 second to make sure that the "start" process does not exit early. WDYT? Any better ideas?